data_steward/cdr_cleaner/cleaning_rules/null_person_birthdate.py

```diff
@@ -0,0 +1,145 @@
+"""
+Null Person Table Birth Date Fields
+
+In the person table, the fields month_of_birth, day_of_birth, and birth_datetime should be nulled.
+The year_of_birth field should remain unchanged.
+
+Original Issue: DC-1356
+"""
+
+# Python imports
+import logging
+
+# Project imports
+import constants.bq_utils as bq_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from common import JINJA_ENV
+from utils import pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+
+PERSON = 'person'
+
+NULL_DATE_QUERY = JINJA_ENV.from_string("""
+SELECT
+    * REPLACE (
+        NULL AS birth_datetime,
+        NULL AS month_of_birth,
+        NULL AS day_of_birth
+    )
+FROM
+    `{{project_id}}.{{dataset_id}}.{{person_table}}` AS per
+""")
+
+
+class NullPersonBirthdate(BaseCleaningRule):
+
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id):
+        """
+        Initialize the class with proper information.
+
+        Set the issue numbers, description and affected datasets. As other tickets may affect
+        this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+        """
+        desc = 'Set Patient Birthdate Fields to NULL'
+        super().__init__(issue_numbers=['DC1356'],
+                         description=desc,
+                         affected_datasets=[cdr_consts.CONTROLLED_TIER_DEID],
+                         affected_tables=PERSON,
+                         project_id=project_id,
+                         dataset_id=dataset_id,
+                         sandbox_dataset_id=sandbox_dataset_id)
+
+    def setup_rule(self, client, *args, **keyword_args):
+        """
+        Load required resources prior to executing cleaning rule queries.
+
+        Method to run data upload options before executing the first cleaning
+        rule of a class.  For example, if your class requires loading a static
+        table, that load operation should be defined here.  It SHOULD NOT BE
+        defined as part of get_query_specs().
+        :param client:
+        :return:
+        """
+        pass
+
+    def get_query_specs(self):
```

Comment:
Forgot this as well, this function signature needs to be updated as well.

```suggestion
    def get_query_specs(self, *args, **keyword_args):
```